test/unit/testreporter.js

```diff
@@ -74,3 +74,5 @@ var TestReporter = function(browser, appPath) {
     setTimeout(sendQuitRequest, 500);
   };
 };
+
+exports.TestReporter = TestReporter;
```

Comment:
I don't really understand this change. There is no `exports` variable in this file. Could you explain this a bit?